### ISSUE TITLE
Installation failure on anaconda Python 3.6

### ISSUE THREAD

**Issue by toqduj** [silx-kit/fabio#116]
Hello. 

It looks like a simple dependency issue, but I haven't been successful in installing fabio in anaconda's 3.6 environment. I only need to be able to read EDF files...

![fabio_anaconda](https://user-images.githubusercontent.com/5449929/27184589-e9e726c2-51e2-11e7-9b6d-4b21c59a96d8.png)

Cheers, 

Brian.

**Comment by vallsv** [comment]
HI Brian,

I am not very familiar with conda, but did you try to use conda-forge?

conda /install/ -c /conda/-/forge fabio/

https://anaconda.org/conda-forge/fabio

BTW, we created a conda package from dev branch one week ago for private
use, and we did not notice any dependency issues.

Let us know

Regards,

**Comment by kif** [comment]
On Thu, 15 Jun 2017 07:01:29 -0700
Valentin Valls <notifications@github.com> wrote:

> BTW, we created a conda package from dev branch one week ago for private
> use, and we did not notice any dependency issues.

The wheels we built should be useable (in Anacond3.6). Are you interested ?

-- 
Jérôme Kieffer
tel +33 476 882 445

**Comment by toqduj** [comment]
Yes, I'd be interested in that. The conda-forge solution indicated the same dependency issues as before. (see screenshot). 

![fabio_anaconda_v2](https://user-images.githubusercontent.com/5449929/27216555-8466ca20-5276-11e7-9287-91398ba1f726.png)

**Comment by toqduj** [comment]
The wheels worked wonderfully, thanks, Jerome!
